Makefile

```diff
@@ -36,7 +36,7 @@ backend: prebuild
 	CGO_ENABLED=0 GOOS=linux go build -ldflags '-w -s -extldflags "-static"' -o $(BUILD_DIR)/backend github.com/opensds/multi-cloud/backend/cmd
 
 s3: prebuild
-	CGO_ENABLED=0 GOOS=linux go build -ldflags '-w -s -extldflags "-static"' -o $(BUILD_DIR)/s3 github.com/opensds/multi-cloud/s3/cmd
+	CGO_ENABLED=1 GOOS=linux go build -ldflags '-w -s -extldflags "-dynamic"' -o $(BUILD_DIR)/s3 github.com/opensds/multi-cloud/s3/cmd
```

Comment:
any specific reason we link dynamically ? how will this affect

1. version changes of other libraries
2. OS vagaries

Reply:
1. it doesn't need the version changes of other libraries
2. because we integrate yig storage backend and yig depends on ceph & librdkafka these libraries, so we need to use the dynamic linking instead of static linking to build s3.